impala/tests/test_util.py

```diff
@@ -78,6 +79,73 @@ def test_get_cookie_expiry(self):
         now = datetime.now()
         assert now + days2k <= get_cookie_expiry({'max-age': '172800000'}) <= now + days2k + sec
 
+    def test_get_matching_cookies(self):
```

Comment:
I can add a case for that. I don't think it will matter, we require exact matches for cookie names. https://stackoverflow.com/a/11312272 suggests that cookie names are case sensitive.

Reply:
No need to add a case.  I was thinking that URL paths were case insensitive, but they are actually case sensitive.